### ISSUE TITLE
Which nvlink port is used for each ring channel?

### ISSUE THREAD

**Issue by anchenchai** [NVIDIA/nccl#655]
hi there,
  As I understand it, nccl will create multiple channels in order to achieve full bandwidth of nvlinks. However, how could I know which nvlink port is used for each channel?  Nvswitch will automatically balance the traffic load among different ports between 2 GPUs?

**Comment by sjeaugey** [comment]
Indeed, when multiple NVLinks are present between two GPUs, all traffic will be equally spread on all NVLinks. So you can see it as a large NVLink connection with a lot of bandwidth, which NCCL might want to use for multiple flows/channels.

**Comment by anchenchai** [comment]
Thanks for your reply.
And for my curiosity, when the version supporting H100 will be released? Since Sharp will be supported by nvswitch inside a node for H100, does it mean that we don't need any cuda kernels for some collective operations under 256 GPUs, for example, we can only use SHARP host APIs to achieve allreduce or broadcast? 
What will be the roadmap for NCCL in future?

**Comment by stephenmsachs** [comment]
### Issue Cleanup: Helping Us Focus on Current Challenges

  We're reviewing older issues to ensure we prioritize the most relevant and active ones. Since this issue hasn't seen updates in over 6 months, we'll be closing it for now.

  ,*This change helps us focus our efforts on addressing any current issues our users are facing.* If this issue still affects you, please don't hesitate to reopen it with a quick update (e.g., "Still relevant on [version=X]").
  Thanks for your understanding and for contributing to NCCL.
